src/graphql/resolvers/box-resolver.ts

```diff
@@ -111,5 +116,12 @@ export class BoxResolver {
       skip,
       take
     });
+
+    return boxes.map((box) => {
+      return {
+        ...box,
+        beingSpent: unconfirmedInputBoxIds.indexOf(box.boxId) > -1
+      };
+    });
```

Comment:
Let's avoid unnecessary mappings by checking if `unconfirmedInputBoxIds` is empty. If so we can return `boxes` directly.